service/src/main/java/uk/gov/hmcts/reform/fpl/enums/HearingBookingKeys.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum HearingBookingKeys {
+    HEARING_DETAILS("hearingDetails");
```

Comment:
minor, now enum does not make a lot of sense, maybe just string constant

Reply:
Yeah enum is a bit overkill, you're right

Reply:
you ok for this to exist publicly in hearing booking service?